CMakeLists.txt

```diff
@@ -43,6 +43,7 @@ endif()
 set(ENABLE_ASTYLE      OFF CACHE BOOL "")
 set(ENABLE_CLANGQUERY  OFF CACHE BOOL "")
 set(ENABLE_UNCRUSTIFY  OFF CACHE BOOL "")
+set(ENABLE_BENCHMARKS  ON  CACHE BOOL "")
```

Comment:
Should this be off or on by default? Any thoughts @white238 ?

Reply:
Off by default is better. I have ran into problems before and its just another to break on customers.